submissions/opeyemiPeter/htmlForms/contactPage.html

```diff
@@ -4,25 +4,90 @@
     <meta charset="UTF-8" />
```

Comment:
Styling this page will come in later on. Revert your changes and remove the images added @opeyemipeter1759

Reply:
@ToluAkin I am sorry, but I don't understand this correction you are pointing out.
Can you be a bit more specific?
Thank you.
which images are talking about